docs/user_guide.md

```diff
@@ -220,27 +220,24 @@ Alpha instance can register to the discovery service, Omega obtains Alpha's inst
 
 ### Spring Cloud Eureka
 
-1. build special version of Eureka
+Uses Spring Cloud Netflix 2.x by default, if you want to use Spring Cloud Netflix 1.x, you can use `-Pspring-boot-1` to switch Spring Cloud Netflix 1.x
 
-   add build parameters `spring-cloud-eureka`
+1. run alpha
 
-   ```bash
-   git clone https://github.com/apache/servicecomb-pack.git
-   cd servicecomb-pack
-   mvn clean install -DskipTests=true -Pspring-cloud-eureka,spring-boot-2
-   ```
-
-   **Note:** Uses Spring Boot 2.x by default, if you want to use omega in the Spring Boot 1.x, you can use `-Pspring-cloud-eureka,spring-boot-1` to switch Spring Boot version to 1.x
-
-2. run alpha
-
-   run with parameter`spring.profiles.active=spring-cloud-eureka`
+   run with parameter `eureka.enabled=true`
 
    ```bash
-   java -Dspring.profiles.active=prd -D"spring.datasource.url=jdbc:postgresql://${host_address}:5432/saga?useSSL=false" -jar alpha-server-${saga_version}-exec.jar --spring.profiles.active=spring-cloud-eureka
+   java -jar alpha-server-${saga_version}-exec.jar \ 
```

Comment:
If there are lots of parameters need to be passed, we need to consider a startup script for it.

Reply:
Is your suggestion run_with_eureka.sh ?

Reply:
yes, it is.